### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 94)

### LINKED ISSUE #1304: [devtools-translate] Portuguese - What's New in DevTools (Chrome 94)
Post: https://developer.chrome.com/blog/new-in-devtools-94/
Hero image: image/dPDCek3EhZgLQPGtEG3y0fTn4v82/kTSL9x5E2JRNenbqZtMO.jpg

Translator: @alvarocamillont
Reviewer: @khaosdoctor

Stable rollout: Sep 21
Translation ready by: Sep 14

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes #1304

Changes proposed in this pull request:

    Add portuguese translations for What's New in DevTools M94

### CODE REVIEW

**Comment by chybie** [comment]
Thanks @alvarocamillont , @khaosdoctor and @andreban, please help to review this.

File: site/pt/blog/new-in-devtools-94/index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/6dedde59f9d64290756a826f73dfe24cf382a470 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/27aa364d1b194a7a778e7649e1f144abbed5957f #}
+
+Issue relacionada: [1179186](https://crbug.com/1179186)
+
+
+## Exibição consistente dos acessores nativos como propriedades no Console {: #native-accessors }
+
+O **Console** agora mostra acessores nativos como propriedades consistentemente.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-94/index.md:112]:
```suggestion
O **Console** agora mostra métodos de acesso nativos como propriedades de forma consistente.
```

File: site/pt/blog/new-in-devtools-94/index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+Por exemplo, ao avaliar a expressão `new Int8Array ([1, 2, 3])` no **Console**, acessadores nativos como `length`,` byteOffset` não eram exibidos na visualização. Com esta atualização, os acessores nativos aparecem na visualização e os valores são avaliados quando expandidos.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-94/index.md:114]:
```suggestion
Por exemplo, ao avaliar a expressão `new Int8Array ([1, 2, 3])` no **Console**, métodos de acesso nativos como `length`,` byteOffset` não eram exibidos na visualização. Com esta atualização, os métodos de acesso nativos aparecem na visualização e os valores são avaliados quando expandidos.
```

File: site/pt/blog/new-in-devtools-94/index.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XVUY8XxbGZW74kPsGkOZ.png", alt="Adequação dos stacks traces para scripts inline com #sourceURL", width="800", height="425" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/c2f30c2b3f637c2339e8b9672c5c59a21b7d1095 #}
+
+Issue relacionadas: [1183990](https://crbug.com/1183990), ​​[578269](https://crbug.com/578269)
+
+## Alteração do formato da configuração da cor no painel Computed  {: #color-unit }
+
+Você agora pode alterar o formato da cor de qualquer elemento no painel Computed segurando a tecla <kbd>Shift</kbd> e clicando  na visualização da cor.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-94/index.md:137]: nit: remove double space.

```suggestion
Você agora pode alterar o formato da cor de qualquer elemento no painel Computed segurando a tecla <kbd>Shift</kbd> e clicando na visualização da cor.
```

File: site/pt/blog/new-in-devtools-94/index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/97143f7586d540e53a2e40ced7f106181e5c9ce3 #}
+
+Issue relacionada: [1226371](https://crbug.com/1226371)
+
+## Substituição dos tooltips customizados por tooltips nativos do HTML {: #tooltip }
+
+O DevTools agora adota tooltips nativos do HTML em todos os componentes. O DevTools teve uma implementação de tooltips customizados por muito tempo devido à falta de estilização de um tooltip nativo.
+
+Infelizmente, manter uma implementação de tooltips customizado é complicado e encontramos regularmente bugs complicados.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-94/index.md:149]: - avoiding repetition of "complicado"
- "encontramos bugs complicados regularmente" seems to read a bit better.
```suggestion
Infelizmente, manter uma implementação de tooltips customizado é complexo e encontramos bugs complicados regularmente.
```

**Review by andreban — COMMENTED** [review]
Thanks for writing this. Left some notes.

File: site/pt/blog/new-in-devtools-94/index.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/).*
+
+{% include 'partials/devtools/en/banner.md' %}
```
Comment (chybie) [inline site/pt/blog/new-in-devtools-94/index.md:20]: replace `/en` with `/pt`

File: site/pt/blog/new-in-devtools-94/index.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+Na guia **Issues**, passe o mouse sobre um problema e clique no menu do problema &nbsp; {% Img src="image/admin/4sdCQbpBaG4MpoHB1J08.png", alt="More", width="4", height="20" %} &nbsp; à direita, selecione **Hide issues like this** para ocultá-lo.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GGJzvwvMYSrkirU44STQ.png", alt="Menu de contexto experimental esconder problema", width="800", height="494" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0200fc96fecec0e209e84c21359ab53393860978 #}
+
+Issue relacionada: [1175722](https://crbug.com/1175722)
+
+{% include 'partials/devtools/en/reach-out.md' %}
```
Comment (chybie) [inline site/pt/blog/new-in-devtools-94/index.md:176]: replace `/en` with `/pt`

File: site/pt/blog/new-in-devtools-94/index.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+{% include 'partials/devtools/en/whats-new.md' %}
```
Comment (chybie) [inline site/pt/blog/new-in-devtools-94/index.md:177]: replace `/en` with `/pt`

**Review by chybie — CHANGES_REQUESTED** [review]

**Review by andreban — APPROVED** [review]
Thanks for this! LGTM from a translation perspective.

File: site/pt/blog/new-in-devtools-94/index.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+  "Utilize o DevTools em seu idioma preferido, novos dispositivos Nest Hub, novo badge de consultas de contêiner CSS e muito mais."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/kTSL9x5E2JRNenbqZtMO.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/).*
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-94/index.md:18]:
```suggestion
*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/). Revisão por [Lucas Santos](https://lsantos.dev)*
```

File: site/pt/blog/new-in-devtools-94/index.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+Por exemplo, você pode digitar  "status-code: 404" para filtrar as requisições de rede com status 404. Habilite a caixa de seleção **Invert** para negar o filtro (mostrar todas as requisições de rede que não estão com status 404).
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/xx1ju91Mu3qflyG6E40W.png", alt="Inverta os filtros de rede", width="800", height="474" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/66878d6044df77ba6264a576483bf5aae6b5f3d9 #}
+
+Issue relacionada: [1054464](https://crbug.com/1054464)
+
+
+## Obsolescência da barra lateral do Console {: #deprecated }
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-94/index.md:85]:
```suggestion
## Depreciação da barra lateral do Console {: #deprecated }
```

File: site/pt/blog/new-in-devtools-94/index.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f34c62f543c29ffd4be95c4e93b453aa34644897 #}
+
+Issue relacionada: [1232937](https://crbug.com/1232937)
+
+
+## Exibição dos cabeçalhos `Set-Cookie` na guia Issues e no painel Network {: #raw-cookies }
+
+O DevTools agora exibe cabeçalhos `Set-Cookie` na guia **Issues**.
+
+Anteriormente, o DevTools não mostrava cookies malformados (cabeçalho `Set-Cookie` incorreto) no painel **Network**. Com o novo filtro `response-header-set-cookie` adicionado ao painel **Network**, os usuários podem filtrar a resposta do cabeçalho` Set-Cookie`. O DevTools vinculará os cabeçalhos `Set-Cookie` na guia **Issues** ao painel **Network**.
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-94/index.md:100]:
```suggestion
Anteriormente, o DevTools não mostrava cookies malformados (cabeçalho `Set-Cookie` incorreto) no painel **Network**. Com o novo filtro `response-header-set-cookie` adicionado ao painel **Network**, os usuários podem filtrar a resposta do cabeçalho `Set-Cookie`. O DevTools vinculará os cabeçalhos `Set-Cookie` na guia **Issues** ao painel **Network**.
```

File: site/pt/blog/new-in-devtools-94/index.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/27aa364d1b194a7a778e7649e1f144abbed5957f #}
+
+Issue relacionada: [1179186](https://crbug.com/1179186)
+
+
+## Exibição consistente dos métodos de acesso nativos como propriedades no Console {: #native-accessors }
+
+O **Console** agora mostra métodos de acesso nativos como propriedades de forma consistente.
+
+Por exemplo, ao avaliar a expressão `new Int8Array ([1, 2, 3])` no **Console**, acessadores nativos como `length`,` byteOffset` não eram exibidos na visualização. Com esta atualização, os métodos de acesso nativos aparecem na visualização e os valores são avaliados quando expandidos.
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-94/index.md:114]:
```suggestion
Por exemplo, ao avaliar a expressão `new Int8Array ([1, 2, 3])` no **Console**, propriedades nativas como `length`,` byteOffset` não eram exibidas na visualização. Com esta atualização, os métodos de acesso nativos aparecem na visualização e os valores são avaliados quando expandidos.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Small translation changes

**Comment by chybie** [comment]
Thanks @khaosdoctor. Over to @alvarocamillont for final check. Let me know when it's good to go!

**Comment by alvarocamillont** [comment]
Done!

**Review by chybie — APPROVED** [review]

**Pull request merged**